docs/topics/cache.txt

```diff
@@ -699,6 +699,10 @@ setting for the middleware.  It can be used like this::
     def my_view(request):
         ...
 
+The ``key_prefix`` may be either a string, or a callable which receives the
+request object and returns the key prefix. This allows you to generate a key
+prefix dynamically. The callable may also return ``None`` to skip caching.
```

Comment:
Maybe add "[This allows you generate a key prefix dynamically], for example for per-user caching." to get people to see possible use cases?

Reply:
See https://github.com/django/django/pull/15553/commits/73de352572bd16773175c2e69e053a679c48f738